shared/hooks/useSearch.ts

```diff
@@ -69,8 +73,10 @@ const useSearch = (documents: SearchResult[], query: string): UseSearchReturn =>
     }
   }, [indexing, search, documents])
 
+  const results: string[] = search.search(normalizedQuery).map(result => result.id)
+
   return {
-    data: normalizedQuery.length === 0 ? documents : (search.search(normalizedQuery) as unknown as SearchResult[]),
+    data: normalizedQuery.length === 0 ? documents : documents.filter(document => results.includes(document.path)),
```

Comment:
🤔 It seems a little extra work to me to map the search results to their IDs and then filter the documents for the ones that have those IDs instead of immediately using the search results. What's the advantage here?

Reply:
We lowercase/normalize the content of the search results above. Using those directly therefore leads to the excerpt of search results being displayed lowercased/normalized.

Reply:
Ah, of course, thanks for the explanation :)